lib/utf8_sanitizer.rb

```diff
@@ -7,17 +7,19 @@ def initialize(app)
   end
 
   def call(env)
-    request = Rack::Request.new(env)
-    parser = RackRequestParser.new(request)
+    parser = RackRequestParser.new(Rack::Request.new(env))
     values_to_check = parser.values_to_check
 
     if invalid_strings(values_to_check)
-      Rails.logger.info(event_attributes(env, parser.request))
+      Rails.logger.info(invalid_utf8_event(env, parser.request))
```

Comment:
You might have seen this already, but reek is complaining about Rails.logger being called twice.

Reply:
Fixed